### PR TITLE
Added Vancouver

### PR DESCRIPTION
**Pull request by davidnudelman**

### CODE REVIEW

File: 2023/AzureVancouver/data.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
```
Comment (noopman) [inline 2023/AzureVancouver/data.json:7]: Please enter the github user id's for any user who is allowed to edit this folder via a PR!

File: 2023/AzureVancouver/data.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Azure Vancouver (VanTUG+)",
+    "Country": "Canada",
+    "Url": "http://vantug.com/",
+    "SessionizeApiId": "{Your sessionize event Id}",
```
Comment (noopman) [inline 2023/AzureVancouver/data.json:14]: Either enter a valid Sessionize, or blank this field!

**Review by noopman — CHANGES_REQUESTED** [review]
I am missing an .MD file with basic information about your community! Please refer to some of the other folders here to figure out who to make one! (Copy paste a file and change to match your community information!)

**Comment by davidnudelman** [comment]
@noopman Done! Thanks for the help

**Comment by noopman** [comment]
> @noopman Done! Thanks for the help

Hey there @davidnudelman ! I could still use resolutions to the review comments I made in your data.json file, specifically that you list any GH handles for the persons - at least yourself - who are valid maintainers of your community folder content!

**Comment by davidnudelman** [comment]
Thank you, this should now be completed. This is all new to me!

**Review by noopman — APPROVED** [review]
This is perfect @davidnudelman , thanks for your diligence! Our review is to ensure your data is right and yields the intended results on the site. Thanks also for your patience!

**Pull request merged**